DrivenIIDevelop/Console-Cowboys
# Patient notes feature

## Description

This pull request adds a new feature that allows physicians to create patient reports and save notes. It includes changes to the scrub_hub_project on the Django side. 

## Changes Made

- Added a new app called patient_notes within the scrub_hub_project folder
- Updated the scrub_hub_project/urls.py to include a URL pattern for the notes feature
- Added patient_notes to the installed apps in settings.py
- The patient_notes app includes additions to models, forms, HTML templates, URLs, and views 

## Testing Instructions
HTML templates have been included for testing data flow.

1. Pull down this branch and apply it locally.
2. Start the development server and navigate to localhost:8000/patients/add.
3. Create a new patient and then go to the patient list
4. Click on a patient to view notes
5. Click view details to see note details or click add note to add a new note
